src/services/creatorStaking/creatorsSpaces.ts

```diff
@@ -0,0 +1,51 @@
+import { gql } from 'graphql-request'
+import Cache from '../../cache'
+import { ONE_HOUR } from '../../constant'
+import { soonsocialGraphQlClient } from '../../constant/graphQlClients'
+
+const creatorsSpacesCache = new Cache('creators-spaces', ONE_HOUR)
+
+type CreatorsSpacesInfo = {
+  spaceIds: string[]
+}
+
+export const GET_CREATORS_SPACES = gql`
+  query getCreatorsSpaces($ids: [String!]) {
+    spaces(where: { id_in: $ids, hidden_eq: false }) {
+      id
+      image
+      hidden
+      name
+      about
+      email
+      linksOriginal
+      ownedByAccount {
+        id
+      }
+    }
+  }
+`
+
+export const getCreatorsSpacesInfo = async ({ spaceIds }: CreatorsSpacesInfo) => {
+  const result = await soonsocialGraphQlClient.request(GET_CREATORS_SPACES, { ids: spaceIds })
```

Comment:
need to update it to subsocial before merge?
but isn't it better to have use env which determines whether to use soonsocial or subsocial?